scripts/ci/pipeline-netcore-runtime.yml

```diff
@@ -65,7 +72,7 @@ jobs:
       packageType: 'sdk'
       version: '2.2.203'
     displayName: Install .NET Core 2.2 latest
-    condition: eq(variables['poolname'], 'Hosted VS2017')
+    condition: or(eq(variables['poolname'], 'Hosted VS2017'),startsWith(variables['poolname'], 'Xamarin-ARM64'))
```

Comment:
I think you may need to bump the .NET Core SDK version on ARM64. It is not officially supported in 2.2 and I found it unusable on my machine. Any recent 3.0 preview should work just fine.